doc-Managing_Providers/topics/Amazon_EC2_Providers_Permissions.adoc

```diff
@@ -5,6 +5,11 @@
 Red Hat recommends using Amazon EC2's _Power User_ Identity and Access Management (IAM) policy when adding Amazon EC2 as a cloud provider in {product-title_short}.
 This policy allows those in the _Power User_ group full access to AWS services except for user administration, meaning a {product-title_short} API user can access all of the API functionality, but cannot access or change user
 permissions.
+ 
+[NOTE]
+====
+When adding an Amazon EC2 provider in {product-title_short} with the intention to use the SmartState Analysis feature, Red Hat recommends using the _Admin_ group.  For situations in which assigning the _Admin_ group is unacceptable, manually create an Amazon EC2 policy role using specific permissions. See xref:amazon-ec2-manual-role-creation[] for more information.
```

Comment:
Would it be ok to delete 'Red Hat recommends" here to be more directive? Otherwise we will need to conditionalize this note for cfme...
One tiny nit - SmartState analysis uses a lowercase 'a' in the Glossary.